svid/jwtsvid/tests/check_svid.c

```diff
@@ -278,7 +203,8 @@ START_TEST(test_jwtsvid_Marshal)
 }
 END_TEST
 
-START_TEST(test_jwtsvid_error5)
+// precondition: invalid jwt token with invalid signature
```

Comment:
I would recommend to add infortation about what is expected in this test (postcondition).
Like an error or success